run_cookiecutter_example.py

```diff
@@ -34,4 +35,6 @@
 p.sendline('')
 
 # Runs until the cookiecutter is done; then exits.
-p.interact()
+#p.interact()  # this fails in the github docs workflow
```

Comment:
Looks like people bumped into it a lot, see https://github.com/actions/runner/issues/241. https://github.com/actions/runner/issues/241#issuecomment-924327172 provides a possible solution to enable `tty`.

Also, flake8 will be happier with a blank space:
```suggestion
# p.interact()  # this fails in the github docs workflow
```